### DISCUSSION TITLE
How we can add CocoaPods to project using cmake?

### DISCUSSION THREAD

**Discussion by arungowda09** [axmolengine/axmol#1093] [Q&A]
I have ported game from **cocos2dx to axmol**. while building for IOS I have to add some **pods** dependency  for **Ads and firebase** . I have added pods without cmake but it gives error while compiling missing framework.

every time after build it get resets with no framework in Xcode.
I need help to add **cocoaPods**  frameworks using camkefile itself instead of adding everytime after running Xcode build command.

**Comment by aismann** [comment]
> I have ported game from **cocos2dx to axmol**. while building for IOS I have to add some **pods** dependency for **Ads and firebase** . I have added pods without cmake but it gives error while compiling missing framework.
> 
> every time after build it get resets with no framework in Xcode. I need help to add **cocoaPods** frameworks using camkefile itself instead of adding everytime after running Xcode build command.

Ok. What you think we can do without any more informations?

**Comment by rh101** [comment]
Just to clarify things here, this issue of adding CocoaPods is entirely related to cmake, and does not have anything to do with Axmol (or cocos2d-x for that matter).  

There doesn't seem to be much info on the web about using CocoaPods with cmake, and from what I've read, it may not even be possible, in which case you will need to find alternative ways to add the dependencies to your project.
